CONTRIBUTING.md

```diff
@@ -10,6 +10,21 @@ of conduct can lead to a ban from the project and a report to GitHub.
 
 You can read the code of conduct [here](https://github.com/hikari-py/hikari/blob/master/CODE_OF_CONDUCT.md).
 
+# Versioning scheme
+
+This project follows the versioning scheme stated by [PEP 440](https://www.python.org/dev/peps/pep-0440/), but with a
+slight modification to the pre-release versioning: we denote all pre-releases with the `.dev{num}` extension.
```

Comment:
this bit about a modification isn't a modification anymore since PEP 440 includes that

Reply:
PEP 440 specifies other types of [pre-releases](https://www.python.org/dev/peps/pep-0440/#pre-releases), which we don't allow. Thought this would clarify that, but I will make the `all` bold if that helps to get the point across

Reply:
So this project doesn't allow beta releases?

Reply:
pretty sure doing every pre-release as a dev release falls under PEP 440 without modifying it at all but ig that's just me

Reply:
Currently, from the strict implementation of HikariVersion (https://github.com/hikari-py/hikari/blob/master/hikari/internal/ux.py#L264-L315) it doesn't. Code there could be modified to support it

My idea of releases has been `MAJOR.MINOR.PATCH.dev{num}` for all beta, alpha and release candidate versions, with `MAJOR.MINOR.PATCH` for the latest versions. Im open to change it tho

Reply:
the lib still does alpha and beta release though it just doesn't explicitly version them and instead marks them all as dev releases which doesn't modify PEP 440, just ignores a part of it which isn't a modification

Reply:
Ok, will remove that part then :+1: